src/contributing/index.md

```diff
@@ -1,13 +1,35 @@
 # Contributing
 
-There's more to running a distribution than just writing code. This section
-explains how to be an active part of Void.
+There's more to running a distribution than just writing code.
 
-Please also visit the Void Web site for further information about [how to
-participate](https://voidlinux.org/contribute/), including our communication
-channels and how to contribute to the Void package repository.
+To contribute to the Void packages repository, start by reading the
+[CONTRIBUTING](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)
+document in the void-packages GitHub repository.
 
-## Section Contents
+To contribute to the Void documentation, start be reading the
+[CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md)
+document in the void-docs GitHub repository.
 
-- [Usage Statistics](./usage-statistics.md)
-- [Contributing To The void-docs Project](./void-docs/index.md)
+If you have any questions, ask them via IRC in #voidlinux on irc.freenode.net,
+or in [the voidlinux subreddit](https://reddit.com/r/voidlinux).
+
+## Usage Statistics
+
+If you would like to contribute usage reports, the
+[PopCorn](https://github.com/the-maldridge/popcorn) program reports installation
+statistics back to the Void project. These statistics are purely opt-in, the
+reporting programs are *not* installed by default on any void systems.
```

Comment:
"are *not* installed nor enabled by default on any Void systems"

"reporting programs" reads a bit weird to me, but idk a better phrasing.

Reply:
Changed to: "These statistics are purely opt-in - PopCorn is *not* installed or enabled by default on any Void systems."